DUnitX.FileUtils.pas

```diff
@@ -0,0 +1,185 @@
+{***************************************************************************}
```

Comment:
Should use the same naming convention as exiting utils units. So should be DUnit.Utils.File.pas